frontend/src/components/mine/Permit/MinePermitInfo.js

```diff
@@ -139,9 +139,7 @@ const transformChildRowData = (amendment, record, amendmentNumber) => ({
 
 export const MinePermitInfo = (props) => {
   const amendmentHistory = (record) => {
-    const childRowData = record.amendments.map((amendment, index) =>
-      transformChildRowData(amendment, record, record.amendments.length - index)
-    );
+    const childRowData = record.amendments.map(transformChildRowData(transformChildRowData));
```

Comment:
Why is `transformChildRowData` being passed to `transformChildRowData`? It doesn't look like this should work or be necessary.

It looks like this should be all that's needed:
```javascript
const childRowData = record.amendments.map(transformChildRowData)
```

Reply:
Not sure why I did that. Changed 👍